### DISCUSSION TITLE
Welcome to MOOgiwara Discussions!

### DISCUSSION THREAD

**Discussion by KOOKIIEStudios** [BAA-Studios/MOOgiwara#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anson1788** [comment]
hi ,i am impressed by this project, i want to try to run it in localhost, but cannot find the database dump, can u provide me thz!

**Reply by Bratah123** [reply]
Hi! you can follow these [instructions](https://github.com/BAA-Studios/MOOgiwara/wiki/MongoDB-Installation#mongodb-community-edition-60) To setting up your local database, there is no database dump because we are using nosql and tables are generated dynamically whilst you run the project!

Hope this helps.

**Comment by anson1788** [comment]
thz, by the way, what mean by mulligan.

**Reply by Bratah123** [reply]
A Mulligan in One Piece TCG is the act of shuffling your hand into the deck, and drawing a new set of cards from the deck.

**Comment by anson1788** [comment]
Thanks for the tip!

**Comment by anson1788** [comment]
by the way , i think i can help developing the project, i am in hong kong ,
do u have any whatsapp / telegram which can share to me so that we can
connect and have a chat.

**Reply by Bratah123** [reply]
Unfortunately, our other team member @KOOKIIEStudios is currently on a break, and he would be closer to your timezone than I would.
We accept and encourage any Pull Requests, to the repo, and any qualified code will be merged. As this is our goal with being open sourced.

**Comment by anson1788** [comment]
No problem, I understand.
Do u know where do he live

在 2023年6月14日週三 上午11:11，Brandon Nguyen ***@***.***> 寫道：

**Reply by KOOKIIEStudios** [reply]
I live in somewhere with the same timezone as you, but not in the same country

**Comment by BBeretta** [comment]
Heey! How are you doing?
I saw that there was no more movement around here, I was following the project and happy with the results...
We look forward to seeing you progress with the project in the future <3

**Reply by Bratah123** [reply]
Unfortunately, the majority of the team has been split apart of the project due to personal issues. I personally fell out of love with OPTCG and don't have real motivation to pursue this project for now. Currently, also finishing up my last year of school and doing internships. I can see this project's framework being used for another card game (Lorcana 😉) in the future, though.

**Reply by BBeretta** [reply]
Got it!! Thanks!!!
